cli/command/formatter/port.go

```diff
@@ -0,0 +1,23 @@
+package formatter
+
+import "github.com/docker/go-connections/nat"
+
+// PortWrite writes the context
+func PortWrite(ctx Context, frontends []nat.PortBinding) error {
+	render := func(format func(subContext subContext) error) error {
+		for _, frontend := range frontends {
+			pCtx := &portContext{PortBinding: frontend}
+			if err := format(pCtx); err != nil {
+				return err
+			}
+		}
+		return nil
+	}
+	subCtx := &portContext{}
+	return ctx.Write(subCtx, render)
+}
+
+type portContext struct {
```

Comment:
@AkihiroSuda thanks for the feedback.
I will add it to the commit but just a small question: from where MarshalJSON is called?
Out of curiosity I tried to find where it's used but failed.
If you can give me some information why we are using it, I will be grateful.

Thanks 😃